pkg/app/web/src/components/copy-icon-button/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { IconButton } from "@material-ui/core";
+import copy from "copy-to-clipboard";
+import { FileCopyOutlined as CopyIcon } from "@material-ui/icons";
+import { FC, useCallback, memo } from "react";
+import { useDispatch } from "react-redux";
+import { addToast } from "../../modules/toasts";
+
+export interface CopyIconButtonProps {
+  name: string;
+  value: string;
+  size?: "small" | "medium";
+  className?: string;
+}
+
+export const CopyIconButton: FC<CopyIconButtonProps> = memo(
+  function CopyIconButton({ name, value, className, size }) {
+    const dispatch = useDispatch();
+    const handleCopy = useCallback(() => {
+      copy(value);
+      dispatch(addToast({ message: `${name} copied to clipboard.` }));
+    }, [dispatch, value, name]);
+
+    return (
+      <IconButton
+        className={className}
+        aria-label={`Copy ${name}`}
+        onClick={handleCopy}
+        size={size}
+      >
+        <CopyIcon fontSize={size === "small" ? "small" : "default"} />
```

Comment:
I guess the type of size should be `"small" | "default"` so that we could remove this ternary operator. (make the default value for this `size` in line 16)

Reply:
@khanhtc1202 `size` is also used by `IconButton`, and is a different type from `fontSize` in `CopyIcon`. 
(`size = 'small' | 'medium' | 'large'` and `fontSize = 'inherit' | 'default' | 'small' | 'large'`)
To support both sizes, I use the ternary operator here.

Reply:
I get your point 👍 I missed line 28 🙏